composer.json

```diff
@@ -3,6 +3,7 @@
     "require" : {
         "swiftmailer/swiftmailer": "v5.0.3",
         "michelf/php-markdown": "1.4.0",
+        "guzzlehttp/guzzle": "4.2.4",
```

Comment:
We'll fix the others separately. Someone should go through and check if there's been any security updates to those libraries too…